test/unit_tests/hoc_python/test_reference_counts.py

```diff
@@ -0,0 +1,281 @@
+import sys
+import neuron
+from neuron import h
+
+
+def test_section():
+    section = h.Section(name='section')
+    assert sys.getrefcount(section) == 2
+
+    # {Py_tp_repr, (void*) pysec_repr_safe},
+    repr(section)
+    assert sys.getrefcount(section) == 2
```

Comment:
Here's where we gain something:
```
    assert sys.getrefcount(section) == base_refcount
```
which one would interpret as `base_refcount + 0`, i.e. it correctly returned to `+0`.